R/write_webtext.R

```diff
@@ -0,0 +1,372 @@
+#' Check hospital reporting data latency and completeness.
+#'
+#' This function retrieves hospital reporting data from
+#' data.cdc.gov and checks for data latency and reporting
+#' completeness issues. It returns a flag string
+#' describing any reporting issues.
+#'
+#' @param reference_date Date, the reference date for the
+#' forecast.
+#' @param disease Character, disease name ("covid" or
+#' "rsv").
+#' @param included_locations Character vector of location
+#' codes that are expected to report. Default:
+#' hubhelpr::included_locations.
+#'
+#' @return Character string describing reporting issues,
+#' or empty string if no issues.
+check_hospital_reporting_latency <- function(
+  reference_date,
+  disease,
+  included_locations = hubhelpr::included_locations
+) {
+  desired_weekendingdate <- as.Date(reference_date) - lubridate::dweeks(1)
```

Comment:
```suggestion
  desired_weekendingdate <- as.Date(reference_date) - lubridate::weeks(1)
```
acc to DHM's suggestion from a different PR